homeassistant/scripts/__init__.py

```diff
@@ -48,6 +50,14 @@ def run(args: List) -> int:
     logging.basicConfig(stream=sys.stdout, level=logging.INFO)
 
     for req in getattr(script, 'REQUIREMENTS', []):
+        try:
+            # Only use the requirement's project_name to verify it exists
+            project_name = Requirement.parse(req).project_name
```

Comment:
Let's not reinvent the wheel. Import `PackageLoadable` from `homeassistant.requirements`